README.md

```diff
@@ -101,7 +101,7 @@ To install the MapboxNavigation framework using [Carthage](https://github.com/Ca
 1. Create a [Cartfile](https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#github-repositories) with the following dependency:
    ```cartfile
    # Latest stable release
-   github "mapbox/mapbox-navigation-ios" ~> 2.0
+   github "mapbox/mapbox-navigation-ios" ~> 2.2
    # Latest prerelease
    github "mapbox/mapbox-navigation-ios" "v2.2.0-rc.1"
```

Comment:
We'll have to update this part when `v2.3.0-alpha.1` is out.

Reply:
So we don't update it right now, but wait for the `v2.3.0-alpha.1` is out? Got it.

Reply:
I think for major versions we were temporarily removing this part, but if `v2.3.0-alpha.1` is going to be out soon I guess it's fine to leave it.